.golangci.yaml

```diff
@@ -69,6 +143,17 @@ issues:
     - path: "_test\\.go"
       linters:
         - wrapcheck
+    - path: tests/e2e_test/
+      linters: [ gci ] # Disable gci due to the test utilities dot import.
+    - path: pkg/watcher/certificate_suite_test.go
+      linters: [ gci ] # Disable gci due to the test utilities dot import.
+    - path: internal/declarative/v2/test/v1/declarative_test.go
```

Comment:
is it possible use `text:` to only exclude the dot import, but allow other gci rules? 
https://golangci-lint.run/usage/configuration/#issues-configuration

Reply:
Unfortunately, the `gci` linting error message is to generic: 

```
File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/kyma-project/lifecycle-manager) -s blank -s dot --custom-order
```

It just complains that the imports are not properly sorted without informing the particular section. So it makes it impossible to filter-out only the dot-imports, using the `text:` filter you suggested.